WebDevStudios/wd_s
# Hotfix/#503 default button colors

Closes #503

### DESCRIPTION ###
- Our default button styles weren't allowing for user selected styles to display on the Front End. This PR will remedy that issue. I also ensured the selected button color remains on hover and the entire button will just get the opacity change.

### SCREENSHOTS ###
![BackEnd](https://www.dropbox.com/s/0de11cbcpoj1uqf/Screenshot%202019-11-01%2015.17.02.png?raw=1)
![FrontEnd](https://www.dropbox.com/s/r7fcdsgcgmy0zgn/Screenshot%202019-11-01%2015.15.40.png?raw=1)

### OTHER ###

### STEPS TO VERIFY ###
Create a new page or post and add a default button. Save the post to ensure there are styles in place if nothing is selected. Then edit the page, change the button styles around to your liking, save the update and confirm the backend matches the front end.
